avaframe/tests/test_geoTrans.py

```diff
@@ -268,3 +270,96 @@ def test_areaPoly(capfd):
     area = geoTrans.areaPoly(x, y)
     tol = 1e-14
     assert area == pytest.approx(A, rel=tol)
+
+
+def test_remeshData(tmp_path):
+    """ test shape of interpolated data onto new mesh """
+
+    headerInfo = IOf.cASCheader()
+    headerInfo.cellsize = 5
+    headerInfo.ncols = 4
+    headerInfo.nrows = 4
```

Comment:
always try to put a different number of rows and cols to also check that we do nor mix up rows and cols

Reply:
I changed the test so that ncols=4, nrows=5